resolwe/process/fields.py

```diff
@@ -593,7 +593,7 @@ def contribute_to_class(self, process, fields, name):
         : param fields: Fields registry to use
         : param name: Field name
         """
-        for field_name in dir(self.field_group):
+        for field_name in self.field_group.__dict__:
```

Comment:
Add a comment why this is changed. You can also include this link: https://www.python.org/dev/peps/pep-0520/#an-order-preserving-cls-dict

Reply:
done